docs/community/configuration.rst

```diff
@@ -0,0 +1,102 @@
+Configuration for compiling GluonNLP website
+-----------------------------------------------------------------
+
+ | The GluonNLP docs website is at https://gluon-nlp.mxnet.io (release branch), or https://gluon-nlp.mxnet.io/master/index.html (master branch). Its source code is at https://github.com/dmlc/gluon-nlp .
+
+Currently the GluonNLP website is constructed from the source code via CI automatically. Nevertheless, if a maintainer wants to make some modification and preview its changes, it will become difficult. I am writing this documentation to note down how to compile the entire website locally for future references.
+
+Website Structure
+~~~~~~~~~~~~~~~~~
+
+Currently the docs part contain four sections: Model Zoo, Examples, API and Community. It should be noted that the model zoo is a link redirecting to the ``scripts`` folder in the parent folder. The other three folders are used exclusively by the docs website. Also, another point that should be paid attention to is that the three different sections use different file format system for compiling - they are inconsistent. So when you work on different sections of the docs website, you should  pay attention to handle the different sections with care.
+
+The main structure, the index file of the entire website, is written in ``rst`` format. It calls the index file of each different section separately. Before compiling the website, you should be aware that: **rst files are static files, they are directly displayed to the website with further styles; md files are script files, the python scripts in these files will be executed and then stored into ipynb files before converting ipynb files into website files**. Or more specifically, the files in the examples folder will be further executed and converted into intermediate files before writing to the final HTML files, while those in other folders don’t need further conversion or computation.
+
+Environment Configuration Instruction
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Next, I will give a step by step instruction on how to compile this website from scratch.
+
+1. Converting md files into ipynb files
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+ | If you don’t modify the examples section, what you need to do here is not converting the md files manually(which is time-consuming), but to get the ipynb files that are already ready for use. They can be found by clicking on the “DOWNLOAD THIS TUTORIAL” button on each page of the examples, like this one: https://gluon-nlp.mxnet.io/master/examples/word_embedding/word_embedding.html . After downloading these files, please put them in the same folder along with each md file. And then *delete* the md files. After these steps, you can skip the remaining of this section and go to the next section.
```

Comment:
1.1 Using existing ipynb from the website
1.2 Generating new ipynb from md files